promtail/templates/config.yml.j2

```diff
@@ -30,8 +29,32 @@ scrape_configs:
           job: fail2ban
           host: {{ ansible_hostname | default('localhost') }}
           __path__: /var/log/fail2ban.log
-        relabel_configs:
-          - source_labels: [__path__]
-            regex: /var/log/fail2ban.log
-            action: keep
-            if: "fileExists('/var/log/fail2ban.log')"
+
+  - job_name: chia
+    static_configs:
+      - targets:
+          - localhost
+        labels:
+          job: chia
+          application: "{{ application }}"
+          component: "{{ component }}"
+          network: "{{ network }}"
+          group: "{{ group_tag }}"
+          ref: "{{ repo_ref }}"
+          host: "{{ ansible_hostname | default('localhost') }}"
+          __path__: "{{ chia_root }}/log/debug.log"
```

Comment:
Any variables that are going to be used in the role need to have defaults (even if an empty string) defined inside the role, so that this role still works even if we aren't using it alongside chia

Reply:
Add default variables